modules/flow-log/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.5.7"
```

Comment:
these versions are different from the rest of the repository but that is intention. It does not affect anything else in the module, but we want to start off on these versions